.github/workflows/cicd-release-validation.yml

```diff
@@ -371,12 +358,46 @@ jobs:
 
   # Run multi-threaded stress tests with 'restart extension' disabled (default behavior)
   # against the kernel mode eBPF sub-system.
-  km_mt_stress_tests:
+  # Stress Tests should run 3x the normal CI/CD workflow
+  # Iteration 1
+  km_mt_stress_tests-1:
     needs: regular
     if: github.event_name == 'workflow_dispatch'
     uses: ./.github/workflows/reusable-test.yml
     with:
-      name: km_mt_stress_tests
+      name: km_mt_stress_tests-1
+      pre_test: .\setup_ebpf_cicd_tests.ps1 -KmTracing $true -KmTraceType "memory"
```

Comment:
Why does this require 3 iterations here rather than making .\execute_ebpf_cicd_tests.ps1 have a parameter to handle it in there?